app/commands/solution/complete.rb

```
@@ -5,7 +5,7 @@ class Complete
     initialize_with :solution, :user_track
 
     def call
-      # TODO: (Required) Guard against already being completed
+      raise SolutionAlreadyCompletedError if solution.completed?
```

@iHiD Do we want to raise an exception or just do an early return?